### PR TITLE
Specify AWS CLI version in devctr-image.md

### PR DESCRIPTION
**Pull request by JonathanWoollett-Light**
## Changes

Added prerequisite to `devctr-image.md` on `aws` version.

## Reason

The instructions given in `devctr-image.md` require AWS CLI 2. Specifically `get-login-password` is not available in version AWS CLI 1. Attempting to run this in AWS CLI 1 results in `aws: error: argument command: Invalid choice, valid choices are:`

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

- [x] All commits in this PR are signed (`git commit -s`).
- [x] If a specific issue led to this PR, this PR closes the issue.
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] New `unsafe` code is documented.
- [x] API changes follow the [Runbook for Firecracker API changes][2].
- [x] User-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

---

- [ ] This functionality can be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md

### CODE REVIEW

**Review by dianpopa — DISMISSED** [review]

File: docs/devctr-image.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ registry. The Firecracker CI suite must also be updated to use the new image.
    you've ever built Firecracker from source.
 1. Access to both an `x86_64` and `aarch64` machines to build the container
    images.
+1. Ensure `aws --version` is >=2.0.0.
```
Comment (pb8o) [inline docs/devctr-image.md:19]: I think maybe this happens with very old versions of v1. My awscli v1 does have it:

```sh
% aws --version
aws-cli/1.25.9 Python/3.8.13 Linux/5.4... botocore/1.27.9

% aws ecr get-login-password
```

Actually it's in the [docs](https://docs.aws.amazon.com/cli/latest/reference/ecr/get-login-password.html)

> This command is supported using the latest version of AWS CLI version 2 or in v1.17.10 or later of AWS CLI version 1.

**Review by pb8o — CHANGES_REQUESTED** [review]

File: docs/devctr-image.md (hunk 1 above)
Comment (JonathanWoollett-Light) [reply docs/devctr-image.md]: Updated to be `>=1.17.10`

**Review by alsrdn — DISMISSED** [review]

**Review by pb8o — APPROVED** [review]

**Review by pb8o — APPROVED** [review]

**Review by luminitavoicu — DISMISSED** [review]

**Review by pb8o — APPROVED** [review]

**Comment by dianpopa** [comment]
@JonathanWoollett-Light can you adjust commit message for this?

**Comment by JonathanWoollett-Light** [comment]
> @JonathanWoollett-Light can you adjust commit message for this?

Updated, looks good?

**Review by dianpopa — APPROVED** [review]

**Review by pb8o — DISMISSED** [review]

**Comment by dianpopa** [comment]
> @dianpopa Updated, looks good?
perfect!

**Review by dianpopa — APPROVED** [review]

**Review by luminitavoicu — APPROVED** [review]

File: docs/devctr-image.md (hunk 1 above)
Comment (roypat) [reply docs/devctr-image.md]: I just tried this on 1.18 (which is what comes preinstalled with Amazon Linux on my c5.metal instance) and it did not find the command `ecr-public` (the docs you link are for the `ecr` command). The docs do not say anything about `ecr-public get-login-password`.

**Pull request merged**